packages/ensnode-sdk/src/api/zod-schemas.ts

```diff
@@ -113,3 +116,52 @@ export const makeRegistrarActionsResponseSchema = (
     makeRegistrarActionsResponseOkSchema(valueLabel),
     makeRegistrarActionsResponseErrorSchema(valueLabel),
   ]);
+
+// Resolution API
+
+/**
+ * Schema for resolver records response (addresses, texts, name)
+ */
+const makeResolverRecordsResponseSchema = (_valueLabel: string = "Resolver Records Response") =>
```

Comment:
remove the `valueLabel` pattern in this file. it's overused in this codebase and not necessary.